### DISCUSSION TITLE
[gatsby-theme-minimal-blog] Changing Font

### DISCUSSION THREAD

**Discussion by yasseen-salama** [LekoArts/gatsby-themes#1121] [Questions]
Hi! I love the Minimal Blog theme, but I want to customise it a little bit and change the fonts.

I installed the plugin [gatsby-plugin-google-fonts-v2](https://github.com/pocorschi/gatsby-plugin-google-fonts-v2) and added it to my Gatsby Config. I went on and  shadowed the `index.js ` file  in the theme-ui folder. I checked that the shadowing is correctly done by changing some colours and it worked :). But, changing the fonts to one of the Google fonts I installed using the mentioned plugin doesn't work.

**Comment by thakursaurabh1998** [comment]
Is this working for you now @yasseen-salama ?
